motor/providers/terraform/provider.go

```diff
@@ -79,22 +85,36 @@ func New(tc *providers.Config) (*Provider, error) {
 		}
 
 		if stat.IsDir() {
-			fileList, err := os.ReadDir(path)
-			if err != nil {
-				return nil, err
-			}
-
-			err = loader.ParseHclDirectory(path, fileList)
-			if err != nil {
-				return nil, errors.Wrap(err, "could not parse hcl files")
-			}
-
-			err = ReadTfVarsFromDir(path, fileList, tfVars)
-			if err != nil {
-				return nil, errors.Wrap(err, "could not parse tfvars files")
-			}
-
-			modulesManifest, err = ParseTerraformModuleManifest(path)
+			filepath.WalkDir(path, func(path string, d fs.DirEntry, err error) error {
+				if err != nil {
+					return err
+				}
+				foundExamples := MODULE_EXAMPLES.FindString(path)
+				if foundExamples != "" {
+					log.Debug().Str("path", path).Msg("ignoring terraform module example")
```

Comment:
if we are ignoring the folder I would log a warning message so it is explicitly clear we skipped that folder